homeassistant/components/camera/push.py

```diff
@@ -117,6 +107,15 @@ def __init__(self, hass, name, buffer_size, timeout, image_field,
         """Call when entity is added to hass."""
         self.hass.data[PUSH_CAMERA_DATA][self.webhook_id] = self
 
+        try:
+            self.hass.components.webhook.async_register(DOMAIN,
+                                                        self.entity_id,
```

Comment:
Why entity id ? This is shown in the UI, let's do name?

Reply:
sorry, you suggested entity name, I read entity id :(